### PR TITLE
data: JeffProd

### PR DESCRIPTION
**Pull request by Tazeg**

### CODE REVIEW

File: data/JeffProd.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/jeffprod",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "YouTube",
+      "url": "https://www.youtube.com/channel/UCAOizN1F23kLohMWo0f27-Q",
+      "icon": "FaYouTube"
```
Comment (khushalarora) [inline data/JeffProd.json:14]: Please replace it with **" FaYoutube "**  :+1: 
```Suggestion
      "icon": "FaYoutube"
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hi @Tazeg, thank you for contributing  :tada: 
Please rename your file as **`Tazeg.json`**
It will look like `data/Tazeg.json`

File: data/JeffProd.json (hunk 1 above)
Comment (Tazeg) [reply data/JeffProd.json]: You need to update your Quick Start picture too : https://user-images.githubusercontent.com/82668196/208292185-743dc114-5699-4b58-bf5c-c5fc74f0248d.png

**Review by iamthecloverly — DISMISSED** [review]

File: data/JeffProd.json (hunk 1 above)
Comment (kumarsonsoff3) [reply data/JeffProd.json]: Hey @Tazeg, we are sorry for the confusion arose at your end, but #5484 is already working on these.

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for your Contribution! 🎉 

<p align="center"><a href="https://linkfree.eddiehub.io/docs/how-to-guides/editing"></a></p>

**Pull request merged**